types/time.go

```diff
@@ -1252,13 +1693,29 @@ func ParseTimeFromFloatString(sc *stmtctx.StatementContext, str string, tp byte,
 	return parseTime(sc, str, tp, fsp, true)
 }
 
+func parseTimeNoFsp(sc *stmtctx.StatementContext, str string, tp byte, isFloat bool) (Time, error) {
```

Comment:
parseTimeNoFsp is almost the same with parseTime ?
Why not provide a parameter and reuse the function?

Reply:
ok

Reply:
Still not addressed

Reply:
I do agree with tiancaiamao.